scripts/post_install/CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 set(KRATOS_GENERATE_PYTHON_STUBS ON CACHE BOOL "Generate hints for binary python packages")
 
-if (${KRATOS_GENERATE_PYTHON_STUBS})
-    if (NOT DEFINED PYTHON_EXECUTABLE)
-        set(PYTHON_EXECUTABLE "python")
-    endif()
+if (${CMAKE_SYSTEM_NAME} MATCHES "Linux") # todo: test the stub generation on other platforms too
```

Comment:
Can we leave out the temporary restriction? @roigcarlo  will check it in windows.